Баланс патч по голопаразитам

ErdGinalD opened the pull request:
Что нового:
- Поросята теперь стоят 14 тк вместо 12
- Поросята теперь таскают без замедления и их нельзя сдвинуть.
- Поросята теперь могут общаться на Общегалактическом(наверное). Нужны тесты. Добавлен новый ТТС некоторых голопарам.

Баланс правки:

- Стандарту снижен урон с 25 до 20. Боевой клич уменьшен с 11 до 8 символов, чтобы клич был в одну строчку.

- Поддержка теперь наносит не 20 токсинов, а 5 + 15 брут/токса. Это сделано для того, чтобы он мог ломать стекла и двери. Подправлено описание абилки.

- Хаосу ЛСД хозяину при телепортации врага заменено на легкие галюны. Для справки, начиная с 20 ударов идут станящие хозяина галюны, так что аккуратнее.

- Молния лишился дамажного луча с руки. Вместо этого, он получил блядскую МОЛНИЮ МАГА! с половиной урона от оригинала, но там все еще крутой дамаг. 

- Защитник теперь не замедляется в защитном режиме, бьет по объектам полноценно(не по людям), получил заклинание тройной стены мага. Стенки не настолько долго держат как у мага, но все еще позволяют держать проход.
TODO: добавить хозяину щит от рига нюки, как только я или кто-то другой получит просветление, как это дерьмо сделать.

- Изменен чат связи стражей, чтобы было четче видно.

## Описание
<!-- Опишите, что делает ваш ПР. Документировать каждую деталь не требуется, просто укажите основные изменения. -->

## Ссылка на предложение/Причина создания ПР
Finally ТЗ от похуя. Джва года ждал.

## Демонстрация изменений
Удивительно, но все работает.

ErdGinalD committed:
Баланс патч по голопаразитам

Что нового:
- Поросята теперь стоят 14 тк вместо 12
- Поросята теперь таскают без замедления и их нельзя сдвинуть.
- Поросята теперь могут общаться на Общегалактическом(наверное). Нужны тесты. Добавлен новый ТТС некоторых голопарам.

ErdGinalD committed:
Еще вдогонку

новый чат, чтобы было заметно читать, голопары теперь не сидят вечность в хозяине.

Rerik007 commented on code/datums/spells/lightning.dm:
```diff
@@ -83,6 +89,28 @@
+	user.Beam(target,icon_state="lightning[rand(1,12)]",icon='icons/effects/effects.dmi',time=5)
+
+	var/energy = min(world.time - start_time,100)
+	if(damaging)
+		Bolt(user,target,max(15,energy/4),5,user) //5 bounces for energy/4 burn
+	else
```
У всего этого прока отличие от родительского только в том что тут делится на 4 а там на 2?
Сделать бы как с damaging хотя бы, тоесть ввести какую нибудь isguardian и уже в lightning/cast проверять /4 или /2. А этот весь lightning/guardian/cast убрать.

ErdGinalD replied:
исправил

Rerik007 commented on code/game/gamemodes/miniantags/guardian/types/healer.dm:
```diff
@@ -72,22 +71,27 @@
 	else
 		to_chat(src, "<span class='danger'>Нужно быть в хозяине для переключения режимов!</span>")
 
+/mob/living/simple_animal/hostile/guardian/healer/AttackingTarget()
+	if(toggle)
+		var/mob/living/L = target
+		if(istype(L))
+			L.adjustToxLoss(15)
+		. = ..()
```
Просто уточняю, бить получается можно будет, только если ты в режиме хила и будет только 15 токсинов. Оно так и задумывалось?

ErdGinalD replied:
дикий проеб, исправил

Rerik007 replied:
А, задумывалось чтобы не делал ничего в хелпе, понял...

Rerik007 commented on code/game/gamemodes/miniantags/guardian/types/standard.dm:
```diff
@@ -15,14 +16,14 @@
 	set name = "Боевой клич"
 	set category = "Guardian"
 	set desc = "Выбери крик при ударе"
-	var/input = stripped_input(src,"Какой ты хочешь боевой клич при ударе? максимум 11 символов.", ,"", 6)
+	var/input = stripped_input(src,"Какой ты хочешь боевой клич при ударе? максимум 8 символов.", ,"", 6)
```
Хорошо конечно, но цифра 6 в конце означает что всегда максимальное количество что она принимало было 5, :roflcat:
 Из-за say("[battlecry][battlecry][battlecry][battlecry][battlecry]", TRUE) на 29 строчке как я понимаю это. Там бы убрать это повторение 5 раз...

ErdGinalD replied:
я четырежды смотрел его код и четырежды не заметил слона в посудной лавке, спасибо за прямой тык куда это исправлять! KEKW

ErdGinalD committed:
fixes

ErdGinalD committed:
now we can pet holopigs

Furrior commented on code/game/gamemodes/miniantags/guardian/types/healer.dm:
```diff
@@ -214,7 +218,7 @@
 					return 1
 		else
-			to_chat(user, "Не удалось обнаружить травму. Возможно, нужно попробовать снова..")
+			to_chat(user, "Проверка окончилась неудачей.")
```
А что не так со старой фразой?

ErdGinalD replied:
вопрос мотивации, по сути. Текущая фраза заставляет по кд жать абилку, а новая четко говорит что провал.

ErdGinalD committed:
добавлена проверка на шокиммун

молния впринципе не дамажит по человеку с шокиммуном, так пусть теперь впринципе не выбирается целью. Закрытие бага
